docs/hw/electrical.md

```diff
@@ -22,6 +22,9 @@ The light ring will show the state of charge and animate while the battery is ch
 !!! attention "Notice"
     Do not attempt to use the robot without its battery installed.
 
+!!! attention "Notice"
```

Comment:
This has gotten pretty cluttered. Why not merge this notice about removing the battery with the previous one? (put line 23 after line 26 and remove line 22)?

Reply:
Agreed. Some images on this page (e.g. how to remove battery) will also help break up the clutter. I'll go ahead and start with this change.